lib/main.js

```diff
@@ -1,9 +1,13 @@
-var _ = require('lodash');
-var css = require('css');
+'use strict';
+
 var cheerio = require('cheerio');
+var postcss = require('postcss');
+var cssWhat = require('css-what');
 var Hashids = require('hashids');
 
-var ALPHABET = 'abcdefghijklmnopqrstuvwxyz';
+var stringify = require('./css-what/stringify');
+
+var ALPHABET = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ';
```

Comment:
You know what, only in quirks mode are they case-insensitive. And knowing email clients, I think it's best to just assume anything can go wrong. Reverting.